config/manager/nephe-controller.yaml

```diff
@@ -55,6 +55,11 @@ spec:
         image: "projects.registry.vmware.com/antrea/nephe:latest"
         imagePullPolicy: IfNotPresent
         name: nephe-controller
+        env:
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
```

Comment:
This will require changes in the helm charts as well.
here: nephe/build/charts/nephe/templates/controller/deployment.yaml

Reply:
Solved.